db/program.go

```diff
@@ -20,6 +20,7 @@ type Program struct {
 	Name        string `firestore:"name" json:"name"`
 	Thumbnail   int64  `firestore:"thumbnail" json:"thumbnail"`
 	UID         string `json:"uid"`
+	WID         string `json:"wid"`
```

Comment:
If you could add a comment here explaining that WID associates the program to a class by WID that would be great. I am planning on improving the internal docs for our types. While we're discussing this, was there any particular reasoning behind using WID over the class' traditional CID?

Reply:
No particular reason, it just seems like we're generally using the WID instead of the CID so I'm just continuing that. Should be trivial enough to switch to the normal CID if you think that's better.

Reply:
Sounds good :+1: